interp/interp_test.go

```diff
@@ -983,6 +991,25 @@ BEGIN { x[1]=3; f5(x); print x[1] }
 	{`{ print $1<$2 }`, "1_0 2", "1\n", "", ""},
 }
 
+func (it interpTest) Flags() map[string]bool {
+	flags := make(map[string]bool)
+	i := strings.Index(it.src, "#")
+	if i < 0 || i+1 == len(it.src) {
```

Comment:
The second clause isn't needed, as `strings.Fields(it.src[i+1:])` will return an empty slice in that case:

```suggestion
	if i < 0 {
```

Reply:
Ok